.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,33 @@
-## Testing
+## ℹ️ Description
 
 <!--
-What is needed to test the changes? e.g. specific cluster, service definitions
-How can one see the result of your work? e.g. configurations, URLs
+What is the change your PR does?
+Please add a short description here. (one sentence is enough)
 -->
 
-## Trade-offs
+- 🔖 JIRA: [DCOS-12345](https://jira.mesosphere.com/browse/DCOS-12345)
+  <!-- add things that reviewers need to know, e.g.
+- 🚫 This blocks #PRNumber
+- ⚠️ Component is not enabled yet (see testing)
+- ⚠️ Wasnt able to test it yet, cluster is still starting… 😅
+  -->
+
+## ⚠️ Dependencies
```

Comment:
Why did you decide that this should go first?

Reply:
> Testing is usually the thing we look for when reviewing - Dependencies and Tradeoffs being below the testing block can easily be overlooked. :/

_PR description_

Reply:
Okay, I might have been unclear here, sorry. I understand your goal, I don't understand how putting it first helps. If I want to just test it and don't care about tradeoffs and dependencies I will just look for the third or fourth headline instead of the first and read from there. 

I think there are better ways to solve the problem, like for example getting a team understanding of why we do PR reviews and what we want to check. We need to get a shared understanding of the thoroughness in which we want to do a PR.

Reply:
As far as I understood these template, we are supposed to remove empty blocks (I did not do it here to show the emojis). 
That means, if there is a Dependency block, than there also is a dependency. -> someone who is always looking for "headline 2" now automatically looks at the dependency block -> mission accomplished :)

Reply:
My assumption for this template basically was that it is a guideline for me creating a PR, helping me adding what I think is necessary and pointing out things that _i think might be_ important. 

that said, I'm free to add a description on top _if I want to_, but lacking a place for adding it, we have these awkward situations where there is text without a headline followed by blocks with headlines. My goal was that I add a place for people describing what they _want_ and not having these weird looking and unpleasing text blocks before headlines. 

being a guideline, the description isnt mandatory. and yes, in the end, this approach (with deleting inapplicable parts) will lead to a somewhat "chaotic" situation where everyone has to "search" for the instructions he is looking for.

Reply:
Sorry, but if it's just about styling I see even less point. By far the most "descriptions" I see are the commit bodies being forgotten to deleted.

I don't like this at all, every descriptive item that should go there has a place or a place can be created, so I would suggest to not add this area. If the team votes for it I will give up and accept this, but I haven't heard an argument that is convincing to me.

Reply:
this comment shouldnt be in this thread I think 🤔

Reply:
It is, I was referring to

>  but lacking a place for adding it, we have these awkward situations where there is text without a headline followed by blocks with headlines.

Reply:
For me, personally, that alone would be reason enough to add this headline…

Reply:
Ok, but there we have different opinions. I fear that people fill out the description and then either leave the other more structured parts alone and you as a reviewer need to pick the information you need out of the description or that we copy stuff over, making it just more text to read for no real reason